metadata-schema/baseSchema.schema.json

```diff
@@ -0,0 +1,350 @@
+{
+  "$id": "/schema/baseSchema",
+  "type": "object",
+  "properties": {
+    "file_reference": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 6,
+      "alternateKeys": [
+        {
+          "tdrFileHeader": "FileReference"
+        }
+      ],
+      "tdrDescription": null
+    },
+    "UUID": {
+      "type": "string",
+      "format": "uuid",
+      "alternateKeys": [
+        {
+          "tdrFileHeader": "FileReference"
+        }
+      ],
+      "message": {
+        "format": "uuid must be a valid UUID"
+      },
+      "Emma Says": "This is a system generated field, not sure what we should say back to users if this fails"
```

Comment:
Can you remove the `Emma Says` from the schema and either add them a comments or record them somewhere more appropriate.